<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerMainWindow</class>
 <widget class="QWidget" name="PlayerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SPYDER Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/spider.png</normaloff>:/icons/icons/spider.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: black;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="Horizontal_splitter">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);

QSplitter::handle
{
    background-color: transparent;
}
</string>
     </property>
     <property name="lineWidth">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,100,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="searchLayout" stretch="0,30">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="Search_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Search All Lists</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:pressed
{
	background-color: rgb(50, 50, 50);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/icons/search.png</normaloff>:/icons/icons/search.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Query_input">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::FocusPolicy::ClickFocus</enum>
               </property>
               <property name="toolTip">
                <string>Enter Search Query (multiple phrases can be searched with + separator.  Example search: sports + 4K)</string>
               </property>
               <property name="styleSheet">
                <string notr="true">/*background-color: rgb(30, 30, 30);
color: rgb(20, 205, 255); */


QLineEdit
{
	border: 1px solid rgb(30, 30, 30);
	background-color: black;
	border-radius: 10px;
	padding-left: 25px;
	color: rgb(20, 205, 255); 
}</string>
               </property>
               <property name="maxLength">
                <number>300</number>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeWidget" name="PlayList_tree">
             <property name="focusPolicy">
              <enum>Qt::FocusPolicy::ClickFocus</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">        QTreeWidget
        {
        background-color: rgb(15, 15, 15);
        background: black;
        color: white;
        }

        QTreeView::branch:open
        {
        image: url(&quot;:/icons/icons/expanded.png&quot;);
        }
        QTreeView::branch:closed:has-children
        {
        image: url(&quot;:/icons/icons/collapsed.png&quot;);
        }

        QTreeWidget::item
        {
        height: 50px;
        }

        QTreeWidget::indicator
        {
        width: 16px;
        height: 16px;
        }

        QTreeWidgetItem::indicator:enabled
        {
        padding-right: 10px;
        }

        QTreeWidget::indicator:checked
        {
        image: url(&quot;:/icons/icons/star-full.png&quot;);
        }

        QTreeWidget::indicator:unchecked
        {
        image: url(&quot;:/icons/icons/star-empty.png&quot;);
        }

        QTreeView::item:selected
        {
        background-color: rgb(30, 30, 30);
        border: 1px solid rgb(82, 26, 149);
        border-left-color: transparent;
        border-right-color: transparent;
        }
        
        QTreeView::item:hover
        {
        background-color: rgb(35, 11, 63);
        border: 1px solid rgb(82, 26, 149);
        border-left-color: transparent;
        border-right-color: transparent;
        }    
            
        QScrollBar:vertical 
        {
            border: 2px solid black;
            background: black;
            width: 15px;
            margin: 22px 0 22px 0;
        }
        QScrollBar::handle:vertical 
        {
            /*border: 1px solid rgb(35, 11, 63);*/
            background: rgb(82, 26, 149);
            min-height: 40px;
            border-radius: 4px; 
        }
        QScrollBar::add-line:vertical 
        {
            border: 2px solid rgb(35, 11, 63);
            background: rgb(82, 26, 149);
            height: 15px;
            border-radius: 4px; 
            subcontrol-position: bottom;
            subcontrol-origin: margin;
        }
        QScrollBar::sub-line:vertical 
        {
            border: 2px solid rgb(35, 11, 63);
            background: rgb(82, 26, 149);
            height: 15px;
            border-radius: 4px; 
            subcontrol-position: top;
            subcontrol-origin: margin;
        }  
        background-color: rgb(15, 15, 15);
        border: 1px solid transparent;</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed|QAbstractItemView::EditTrigger::SelectedClicked</set>
             </property>
             <property name="indentation">
              <number>25</number>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
             <property name="animated">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="bottomhorizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="Settings_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>30</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Settings</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:pressed
{
	background-color: rgb(50, 50, 50);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/icons/icons/settings-sliders.png</normaloff>:/icons/icons/settings-sliders.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="Status_label">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">/*background-color: rgb(30, 30, 30);
color: rgb(20, 205, 255);  */

QLabel
{
	border: 1px solid rgb(30, 30, 30);
	background-color: black;
	border-radius: 10px;
	padding-left: 0px;
	color: rgb(20, 205, 255); 
}</string>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="indent">
                <number>30</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="Vertical_splitter">
         <property name="styleSheet">
          <string notr="true">QSplitter::handle
{
    background-color: transparent;
}</string>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>1</number>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <widget class="QWidget" name="VideoView_widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="ShowControlPanel_left_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>1</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background-color: black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="ShowControlPanel_top_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background-color: black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="ShowControlPanel_bottom_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background-color: black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="ShowControlPanel_right_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>1</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel
{
	background-color: black;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="botomverticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
